comment/models/flags.py

```diff
@@ -12,12 +15,23 @@
 
 
 class Flag(models.Model):
+    UNFLAGGED = 1
+    FLAGGED = 2
+    REJECTED = 3
+    RESOLVED = 4
+    STATES_CHOICES = [
+        (UNFLAGGED, 'Unflagged'),
+        (FLAGGED, 'Flagged'),
+        (REJECTED, 'Flag rejected by the moderator'),
+        (RESOLVED, 'Comment modified by the author'),
```

Comment:
Earlier gettext_lazy was used with the raw message for the state. It would help users from non-english language to easily port this in their language.

```python
from django.utils.translation import gettext_lazy as _

STATE_CHOICES = [
        (UNFLAGGED, _('Unflagged')),
        (FLAGGED, _('Flagged')),
        (REJECTED, _('Flag rejected by the moderator')),
        (RESOLVED, _('Comment modified by the author')),
]

Reply:
You are right. I will open an issue for translating to cover everything.